### PR TITLE
Adds More Wiki Content From Old Wiki

### PR DESCRIPTION
**Pull request by RoboticMind**
Brings in pages from the github wiki and adds some styling fixes for markdown. 

Split this from my ongoing work on the RPC wiki page update so it's not as big of a PR.

### CODE REVIEW

File: Wiki/Addnodes.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Last updated: Mon Oct 28 13:54:39 UTC 2019
+
+Windows: `C:\Users\<username>\AppData\Roaming\GridcoinResearch`
+
+macOS:   `/Users/<username>/Library/Application Support/GridcoinResearch`
+
+Linux:   `/home/<username>/.GridcoinResearch`
+
+### Mainnet
```
Comment (ShmoogleOsukami) [inline Wiki/Addnodes.md:21]: This does not format correctly on build. | should technically become a table but this does not happen currently. This seems limited to this page as the scraper displays correctly.

File: Wiki/Config-File.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    ## Manually Set-up Ports
+    #upnp=false
+    #externalip=<Your IP Address>
+
+## Addnodes
+
+The list of addnodes you provide are the nodes that your client will
+attempt to establish outbound connections with. The basic configuration
+file does not include addnodes. A full current list of addnodes can be
+found [List of Addnodes](List-of-Addnodes "wikilink")
```
Comment (ShmoogleOsukami) [inline Wiki/Config-File.md:105]: Wiki links currently also don't work correctly.

File: Wiki/Addnodes.md (hunk 1 above)
Comment (RoboticMind) [reply Wiki/Addnodes.md:21]: Fixed that, don't know how I missed it. The tables have to be separated by newlines on both sides of the tables and these weren't. I also noticed that the table had something that was supposed to span the column which doesn't work on kramdown so I changed it to be headers above the tables

File: Wiki/Config-File.md (hunk 2 above)
Comment (RoboticMind) [reply Wiki/Config-File.md]: Fixed those links

**Comment by RoboticMind** [comment]
Oh oops I was trying to get git to ignore my work on the RPC file, but I guess I told it to delete the file instead. That was not intentional

**Comment by RoboticMind** [comment]
That should be fixed now. Hopefully I didn't mess anything up when I changed that commit to not delete the RPC page. 

Also as a note about linking to these pages. I plan to add something that does later on in the future, I just want to get these pages working for right now and make sure it looks right

**Pull request merged**